scipy/stats/_distn_infrastructure.py

```diff
@@ -1857,6 +1857,8 @@ def _mom_integ1(self, q, m, *args):
         return (self.ppf(q, *args))**m
 
     def _mom1_sc(self, m, *args):
+        if np.any(np.array(args) >= m):
+            return np.nan
```

Comment:
Thanks for submitting this!
Is this appropriate for `rv_continuous`, or should this be an override for `kappa3`? It makes sense for `kappa3`, but why is it ok for all distributions?

Reply:
> It makes sense for kappa3, but why is it ok for all distributions?

Yeah, that makes sense. In this case, I think overriding seems a better option